Dockerfile

```diff
@@ -25,7 +24,7 @@ RUN go mod download
 # With the above commmand,
 # TARGETOS can be "linux", TARGETARCH can be "amd64", "arm64", and "arm", TARGETVARIANT can be "v7".
 
-ARG TARGETPLATFORM TARGETOS TARGETARCH TARGETVARIANT
+ARG TARGETPLATFORM TARGETOS TARGETARCH TARGETVARIANT VERSION=dev
```

Comment:
Do we need to also update the workflow yaml file to set the `VERSION` arg?
https://github.com/actions-runner-controller/actions-runner-controller/blob/master/.github/workflows/publish-arc.yaml

Reply:
nvm... `build.Version=${VERSION}` will read from the ENV.

Reply:
@TingluoHuang Ahhh, that's actually a good catch!
Although our go-build snippet refers to the VERSION env, I don't think we now pass that env as a build-arg for the docker build.
Probably we might need to add something like `build-args: NPM_TOKEN=${{env.VERSION}}` to the docker-build-push action step.

Reply:
Addressed in [da14999](https://github.com/actions-runner-controller/actions-runner-controller/pull/1659/commits/da149994f51d794159cbb91be865002aff035883)